test/properties.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var jsc = require('jsverify');
```

Comment:
This library and the `jsverify` docs use `jsc` instead of `jsv`. Is that a reference to `JSCheck`?